js2.5.1.js

```diff
@@ -0,0 +1,18 @@
+Array.prototype.square = function() {
+    return this.map(v => v * v);

```

Comment:
change all parameters name which will be related to function and examples has to be included